lib/cartopy/crs.py

```diff
@@ -1184,7 +1184,6 @@ def __init__(self, central_longitude=0.0, central_latitude=0.0,
                                                         globe=globe)
 
         a = np.float(self.globe.semimajor_axis or WGS84_SEMIMAJOR_AXIS)
-        b = np.float(self.globe.semiminor_axis or WGS84_SEMIMINOR_AXIS)
         lon, lat = central_longitude + 180, - central_latitude + 0.01
```

Comment:
@pp-mo And also what's happened to b (the semiminor axis)?

Reply:
'b' also not used anywhere.
As it was identical, I suspect this one was a mistake from the first.

Reply:
Identical to what?

Reply:
Identical to 'a'.
If it was going to be useful, I'd expect it to be different in some way, but it's all just guesswork.

Reply:
It's not identical to 'a', though. But I'm not sure it needs to be used anyway.

Reply:
Oh I see, it is not after all : more care required !